### PR TITLE
The fortnite team member down sound effect now plays when getting sent to crit

### PR DESCRIPTION
**Pull request by MCterra10**
## About The Pull Request
title

## Why It's Good For The Game
title

## Changelog
:cl:
add: The fortnite team member down sound effect now plays when getting sent to crit
/:cl:

### CODE REVIEW

**Comment by cacogen** [comment]
*floss

**Comment by Jordie0608** [comment]
vile_filth

**Comment by MCterra10** [comment]
this is a very high quality sound sourced directly from the game files with none of that background noise found in youtube "sound effect" videos. Only the best in quality for /tg/station

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -754,10 +754,12 @@
 		return
 	if(stat != DEAD)
 		if(health <= HEALTH_THRESHOLD_DEAD && !HAS_TRAIT(src, TRAIT_NODEATH))
+			do_fortnite()
```
Comment (AffectedArc07) [inline code/modules/mob/living/carbon/carbon.dm]: *default dances*

**Comment by waterwater1212** [comment]
+1

**Comment by karmaisblackandbluepilled** [comment]
hey terra mind if I steal this

**Comment by MCterra10** [comment]
yes

**Comment by MCterra10** [comment]
I thought you were banned from the repo anyway

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
Add this to health sensors instead

**Comment by 81Denton** [comment]
we like fortnite

**Comment by CPTANT** [comment]
Isn't this a copyright issue?

**Comment by 81Denton** [comment]
CPTANT is right, if this gets merged Tim Sweeney is gonna run me over with the battle bus

**Comment by AffectedArc07** [comment]
aw shame

**Comment by trollbreeder** [comment]
leave this meme PR for april fools testmerging
please

PLEASE

**Comment by 81Denton** [comment]
You can already test merge closed PRs my man

**Pull request closed without merging**